### ISSUE TITLE
The commondir option no longer appears to have any effect

### ISSUE THREAD

**Issue by appden** [substack/node-browserify#1075]
There are references to `commondir` in the docs and in the code, but it doesn't appear to be used at all anymore.

**Comment by brettz9** [comment]
Yes, it appears it should either be removed or renamed (though it does have an impact on potentially breaking source maps per #895 ).
